client/src/api/ProjectApiService.js

```diff
@@ -52,42 +52,28 @@ class ProjectApiService {
     console.log('THIS BASEPROJECT URL', this.baseProjectUrl);
 
     try {
-      const proj =  await fetch(this.baseProjectUrl, requestOptions);
-      const projectDetails = await proj.json()
-      return projectDetails._id
+      const proj = await fetch(this.baseProjectUrl, requestOptions);
+      const projectDetails = await proj.json();
+      return projectDetails._id;
     } catch (error) {
       console.error(`Add project error: `, error);
       alert('Server not responding.  Please try again.');
       return undefined;
     }
   }
 
-  async updateProject(projectId, fieldName, fieldValue) {
-    let updateValue = fieldValue;
-    // These field are arrays, but the form makes them comma separated strings,
-    // so this adds it back to db as an arrray.
-    if (
-      fieldValue &&
-      (fieldName === 'partners' || fieldName === 'recruitingCategories')
-    ) {
-      updateValue = fieldValue
-        .split(',')
-        .filter((x) => x !== '')
-        .map((y) => y.trim());
-    }
-
+  async updateProject(projectId, projectData) {
     // Update database
     const url = `${this.baseProjectUrl}${projectId}`;
     const requestOptions = {
       method: 'PATCH',
       headers: this.headers,
-      body: JSON.stringify({ [fieldName]: updateValue }),
+      body: JSON.stringify({ ...projectData }),
```

Comment:
If I'm understand this correclty, this change is because we're now submitting a whole JSON object to update the database entry intead of one specific field value.